### ISSUE TITLE
Write our own require-from-string

### ISSUE THREAD

**Issue by danmademe** [express-vue/express-vue-renderer#11]
I think to support the import statements and require statements inside vue files, we need to write our own require-from-string

currently we can't really use imports/requires inside vue objects due to require-from-string actually requiring vue objects.

we need to block this and have it not do this.. so that vue.js will import the .vue files and MAAAYBE this will solve the issue.

## Potential solutions
basically, the functionallity needs to be this .. check if there was a require/import statement, check if it refers to a vue file.. if it does, then run it through the templateCompiler, and then import it in the right way.. 

If its a javascript file, just import it and hope it works..

**Comment by danmademe** [comment]
Just added a PR to move the require functionality to our own project in #12

**Comment by danmademe** [comment]
The whole functionality of this library is like based
